storage/src/backend/registry.rs

```diff
@@ -256,7 +254,6 @@ impl RegistryState {
                 Some(ReqBody::Form(form)),
                 &mut headers,
                 true,
-                true,
             )
             .map_err(|e| einval!(format!("registry auth server request failed {:?}", e)))?;
         let ret: TokenResponse = token_resp.json().map_err(|e| {
```

Comment:
Where is the `token_resp.status()` checked?

Reply:
Thanks for pointing it out. It is handled now